storage/cleanup.go

```diff
@@ -147,7 +158,19 @@ func (m *cleanupManager) deleteInactiveNamespaceFiles() error {
 	return m.deleteInactiveDirectoriesFn(dataDirPath, namespaceDirNames)
 }
 
-func (m *cleanupManager) deleteInactiveFilesetFiles() error {
+// deleteInactiveDatafiles will delete data files for shards that the node no longer owns
+// which can occur in the case of topology changes
+func (m *cleanupManager) deleteInactiveDatafiles() error {
```

Comment:
super nit: rename to `deleteInactiveDataFiles` to stay consistent with other case usage in this diff

Reply:
not sure I'm following this comment.

`deleteInactiveFilesetFiles` is the generic one that is called by `deleteInactiveDatafiles` and `deleteInactivSnapshotFiles`

Reply:
lol i mean `s/deleteInactiveDatafiles/deleteInactiveDataFiles/g`

Reply:
done